eidas-saml-parser/src/test/java/uk/gov/ida/notification/eidassaml/apprule/IstioHeaderTest.java

```diff
@@ -31,6 +32,9 @@
     private static final DropwizardClientRule metadataClientRule = createTestMetadataClientRule();
     private static final EidasSamlParserAppRule eidasSamlParserAppRule = createEidasSamlParserRule(metadataClientRule);
 
+    @ClassRule
+    public static final DropwizardClientRule metatronService = createInitialisedClientRule(new TestMetatronResource());
```

Comment:
Is this actually being used? I'm trying to run the test but intellij is unhappy.